### PR TITLE
QuickSort.cs migration to .NET core

### PR DESCRIPTION
**Pull request by Miggleness**
This is an initial PR for migrating sample code to .NET core, starting with QuickSort.cs.  Do review @rikace  and let me know if this is aligned with what you have in mind for the migration.  I'll be carrying this on as I go thru the book.

### CODE REVIEW

File: Chapter.01/QuickSort.cs/QuickSort.cs.csproj [hunk 1]
```diff
@@ -1,76 +1,12 @@
...
-    <ProjectGuid>{BB41293F-CA44-4AE9-8198-AB1F2A0D71A5}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>QuickSort.cs</RootNamespace>
-    <AssemblyName>QuickSort.cs</AssemblyName>
-    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <TargetFrameworkProfile />
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```
Comment (rikace) [inline Chapter.01/QuickSort.cs/QuickSort.cs.csproj]: What I would do is to enable multi framework targeting.
for example ```<TargetFrameworks>net471; netcoreapp2.1</TargetFrameworks>```

**Comment by rikace** [comment]
Looks good, great work!!
I will create a ```netcore``` branch to work on the conversation in parallel from the original one

**Comment by Miggleness** [comment]
Copy on targeting multiple framework. I agree about having a separate branch so your other readers staying on pre-.NET 4.7/core can continue following the book as is.

**Comment by rikace** [comment]
Work in progress is coming for a complete conversion

**Pull request closed without merging**